test/OpenIddict.Server.Tests/OpenIddict.Server.Tests.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;net472;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Moq" Version="4.13.1" />
```

Comment:
Please use `$(MoqVersion)` to avoid hardcoding the Moq version.

Reply:
Good to know, updated!